driver/modern_bpf/helpers/store/auxmap_store_params.h

```diff
@@ -397,7 +397,14 @@ static __always_inline void auxmap__store_execve_args(struct auxiliary_map *auxm
 		total_len += arg_len;
```

Comment:
```suggestion
		total_len += arg_len;
		if(total_len >= MAX_PROC_ARG_ENV)
		{
			break;
		}
```

If the verifier allows us to do that, it would be great! Becuase we are in some way vulnerable to a sort of excessive length attack. If the userspace sends args for more than `2^16` byte we have an overflow of `total_len` and so we cannot detect the real `args` or `env`. WDYT?

Reply:
After a quick search appears that
```
#define ARG_MAX       131072	/* # bytes of args + environ for exec() */
```
from the kernel 2.6.19. Given that, I think we can just stick with my solution for the sake of simplicity. Furthermore, with the `break`, an attacker could easy hide a large argument and make the driver slower passing a large number of args. In my opinion the best trade off is mine. WDYT?